app/controllers/api/v5/sections_controller.rb

```diff
@@ -3,4 +3,19 @@ def index
     filter_model Section
     filter_any :id, :course_id, :name, :crn
   end
+  def update
+    Section.find(params[:id]).tap{ |section| section.update!(section_params)}
```

Comment:
see: http://ruby-doc.org/core-2.4.0/Object.html#method-i-tap. You shouldn't need that call here. You should be able to just chain `Section.find(params[:id])` with `update!(section_params)`